src/RequirementChecker/Checker.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+/*
+ * This file is part of the box project.
+ *
+ * (c) Kevin Herrera <kevin@herrera.io>
+ *     Théo Fidry <theo.fidry@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace KevinGH\Box\RequirementChecker;
+
+/**
+ * The code in this file must be PHP 5.3+ compatible as is used to know if the application can be run.
+ *
+ * @private
+ */
+final class Checker
+{
+    /** @private */
+    const REQUIREMENTS_CONFIG = '.requirements.php';
+
+    /**
+     * @return bool
+     */
+    public static function checkRequirements()
+    {
+        $requirements = self::retrieveRequirements();
+
+        $checkPassed = $requirements->evaluateRequirements();
+
+        $io = new IO();
+
+        self::printCheck(
+            $checkPassed,
+            new Printer(
+                $io->getVerbosity(),
+                $io->hasColorSupport()
+            ),
+            $requirements
+        );
+
+        return $checkPassed;
+    }
+
+    public static function printCheck($checkPassed, Printer $printer, RequirementCollection $requirements)
+    {
+        if (false === $checkPassed && IO::VERBOSITY_VERY_VERBOSE > $printer->getVerbosity()) {
```

Comment:
FQC should be used for functions in this class since constants and classes can't be imported with use statements